### DISCUSSION TITLE
eye blink

### DISCUSSION THREAD

**Discussion by fusionsoftPSW** [vladmandic/face-api#136] [Q&A] [answered]
is possible detecting eye closed?

**Comment by vladmandic** [comment, marked as answer]
not in `face-api`, but its implemented in my newer library <https://github.com/vladmandic/human>

**Reply by psw7420fusion** [reply]
thanks for creating good library
i tried detecting eye blink by tensorflow KNN Classifier
and success to check eye blink 

Thank you
